modules/ossm-jaeger-supported-configurations.adoc

```diff
@@ -0,0 +1,45 @@
+////
+This REFERENCE module included in the following assemblies:
+-assembly.adoc
+////
+
+[id="ossm-jaeger-supported-configuration_{context}"]
+= Jaeger supported configuration
+
+For the {ProductVersion} release of {ProductName}the only supported configuration of Jaeger is as follows:
+
+.Supported Jaeger configurations
+[options="header"]
+[cols="l, a"]
+|====
+|Spec|Notes
+|strategy: production
+storage:
+    type: elasticsearch
+
+|Elasticsearch is the only supported storage option for this release.  And only when using the elasticsearch-operator to self-provision an Elasticsearch cluster.
+
+|strategy: allinone
+  storage: memory
+|The allinone strategy is available in this release but it is *NOT* supported for production.  This strategy should only be used for development, demo, and test environments due to the fact that data in memory will not be retained.
+
+|kind: Deployment
+metadata:
+  name: myapp
+  annotations:
+    "sidecar.jaegertracing.io/inject": "true"
+spec:
+  selector:
+    matchLabels:
+      app: myapp
+  template:
+    metadata:
+      labels:
+        app: myapp
+|You must opt in to automatic injection of the sidecar.
+
+
+|agent:
+   strategy: DaemonSet
+|DaemonSet is only supported for single tenant installations.
```

Comment:
Need to say that the default is Sidecar, and the only option supported in a multitenant installation.